src/framework/_typography.scss

```diff
@@ -5,6 +5,14 @@
 /** @type {Boolean} Do we need classes? */
 $has-classes: false !default;
 
+/** @type {String} Root font size, used to calculate others font sizes */
+$font-size-base: 16px !default;
+
+// Set root font size
+html {
```

Comment:
This declaration should be wrapped in a `@if $has-classes { ... }` condition to avoid duplicates in the compiled CSS.